src/modules/services_manager_service.rb

```diff
@@ -150,48 +164,32 @@ def reset
       true
     end
 
-    # Returns only enabled services, the rest is expected to be disabled
+    # Returns services to be exported to AutoYast profile
+    #
+    # @return [Hash{String => Array<String>}]
     def export
-      enabled_services = services.select do |service_name, properties|
-        enabled(service_name) && properties[:loaded] && can_be_enabled(service_name)
-      end
+      enabled_services  = exportable_enabled_services.keys | ServicesProposal.enabled_services
+      disabled_services = exportable_disabled_services.keys | ServicesProposal.disabled_services
 
-      # Only services modifed by the user to be disabled are exported
-      # to AutoYast profile, untouched services are not exported
-      disabled_services = services.select do |service_name, properties|
-        !enabled(service_name) && properties[:modified]
-      end
-
-      export_enable = enabled_services.keys | ServicesProposal.enabled_services
-      export_disable = disabled_services.keys | ServicesProposal.disabled_services
-
-      log.info "Export: enabled services: #{export_enable}, disabled services: #{export_disable}"
+      log.info "Export: enabled services: #{enabled_services}, disabled services: #{disabled_services}"
 
-      {
-        'enable' => export_enable,
-        'disable' => export_disable,
-      }
+      { "enable" => enabled_services, "disable" => disabled_services }
     end
 
     def import(profile)
```

Comment:
NP: we can take the opportunity to add some documentation to this `#import` method.